libobs-d3d11/d3d11-subsystem.cpp

```diff
@@ -298,12 +288,9 @@ gs_swap_chain::gs_swap_chain(gs_device *device, const gs_init_data *data)
 	if (flags & DXGI_SWAP_CHAIN_FLAG_FRAME_LATENCY_WAITABLE_OBJECT) {
 		ComPtr<IDXGISwapChain2> swap2 = ComQIPtr<IDXGISwapChain2>(swap);
 		hWaitable = swap2->GetFrameLatencyWaitableObject();
-		if (hWaitable) {
-			hr = swap2->SetMaximumFrameLatency(40);
-			if (FAILED(hr))
-				throw HRError("Could not relax frame latency",
-					      hr);
-		}
+		if (FAILED(hr))
```

Comment:
Is this intentional to check `hr`, not `hWaitable`?

Reply:
It is not. Thanks.